app/_hub/kong-inc/response-transformer-advanced/index.md

```diff
@@ -78,12 +78,12 @@ params:
     - name: append.if_status
       required: false
       description: List of response status codes or status code ranges to which the transformation will apply. Empty means all response codes
-    - name: whitelist.json
+    - name: allow.json
       required: false
       default:
       value_in_examples:
       description: |
-        Set of parameter names. Only allows whitelisted parameters in the JSON response body.
+        Set of parameter names. Only allows allowed parameters in the JSON response body.
```

Comment:
@murillopaula I assume instead of updating the current index, which represents 1.3.x, we should create a 1.3.x version that does not include this information, and create a new index file that does. I can assist with this. It looks like version files were note created for 1.3 and 1.5 releases. The version of the plugin is 0.4.0 as noted in the Lua handler file. Same goes for the other plugins in this PR.

From my notes I'm compiling:

Issues: Missing 1.5.x version, did not create 2.1.x version. Do not know what changed between those past releases.

ResponseTransformerHandler.VERSION = "0.4.0"